recipes/biophi/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "1.0.1" %} # Remember to update sha256 below
+
+package:
+  name: biophi
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/Merck/BioPhi/archive/v{{ version }}.tar.gz
+  sha256: 'dbd27a6eecfbf1df23042b4ea8c87407d201bfa6aad780b44ce51569bff60da9'
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - biophi = biophi.common.cli.main:main
+  script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - pip
```

Comment:
Does it need pip and pytest to run?

Reply:
Good catch, I removed pip from here, and I moved pytest to host section